tests/js/spec/components/events/contextSummary.spec.jsx

```diff
@@ -132,3 +132,43 @@ describe('ContextSummary', function() {
     });
   });
 });
+
+describe('OsSummary', function() {
+  describe('render()', function() {
```

Comment:
This extra `describe` block is not needed.

Reply:
What's the general pattern here, I noticed these `renderer()` describes in a lot of other tests. Would rather keep it consistent than changing it in this one location.

Reply:
Yeah we need to get rid of them all, there's no reason for them, just adds extra nesting.